packages/ui/src/examples/E2_FetchWithStrapiRequest.tsx

```diff
@@ -1,17 +1,37 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { useEffect, useState } from 'react'
 
-import { Box } from '@chakra-ui/react'
+import { Box, OrderedList, ListItem } from '@chakra-ui/react'
 
 import { strapiRequest } from '@wsvvrijheid/lib'
+import { Blog } from '@wsvvrijheid/types'
 
 export const FetchWithStrapiRequest = () => {
   const [blogs, setBlogs] = useState([])
```

Comment:
```suggestion
  const [blogs, setBlogs] = useState<Blog[]>([])
```

You can indicate the type of state whenever possible.